hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeIntegrationTest.java

```diff
@@ -56,5 +56,23 @@ public void testMapCount() {
         verify(1, getRequestedFor(urlPathEqualTo("/ping")).withQueryParam("mpct", equalTo("2")));
 
     }
+
+    @Test
+    public void testMapCountWithBackupReadEnabled() {
+        HazelcastInstance hz = createHazelcastInstance();
+        Node node = getNode(hz);
+        PhoneHome phoneHome = new PhoneHome(node, "http://localhost:8080/ping");
+        Map<String, String> map1 = hz.getMap("hazelcast");
+        node.getConfig().getMapConfig("hazelcast").setReadBackupData(true);
```

Comment:
I think it would make sense to test more things in one integration test, so please rename `testMapCount()` and change it to cover this metric as well. Partly because it is faster (integ test setup and tear-down can be significantly slow), and partly because that way the integ test is closer to what happens in reality.